src/elements/content-sidebar/__tests__/ContentSidebar.test.js

```diff
@@ -1,12 +1,17 @@
 import React, { act } from 'react';
 import noop from 'lodash/noop';
 import { mount } from 'enzyme';
-import { SIDEBAR_FIELDS_TO_FETCH } from '../../../utils/fields';
+import { SIDEBAR_FIELDS_TO_FETCH, SIDEBAR_FIELDS_TO_FETCH_ARCHIVE } from '../../../utils/fields';
 import { ContentSidebarComponent as ContentSidebar } from '../ContentSidebar';
 import SidebarUtils from '../SidebarUtils';
+import { isFeatureEnabled } from '../../common/feature-checking';
```

Comment:
Minor, but we generally try to alphabetize our imports after grouping by third and first-party.

Reply:
Fixed, but I also moved down
```
import { SIDEBAR_FIELDS_TO_FETCH, SIDEBAR_FIELDS_TO_FETCH_ARCHIVE } from '../../../utils/fields';
```